src/main/kotlin/org/wycliffeassociates/resourcecontainer/ZipAccessor.kt

```diff
@@ -0,0 +1,92 @@
+package org.wycliffeassociates.resourcecontainer
+
+import java.io.File
+import java.io.IOException
+import java.io.BufferedReader
+import java.io.BufferedWriter
+import java.io.FileOutputStream
+import java.util.zip.ZipEntry
+import java.util.zip.ZipFile
+import java.util.zip.ZipOutputStream
+
+class ZipAccessor(private val file: File) : IResourceContainerAccessor {
+    private var _zipFile: ZipFile? = null
```

Comment:
don't make this nullable

Reply:
I don't want a new ZipFile upon creation of a ZipAccessor, because I don't want the file created on disk yet.  (Deferring that allows the first call to `write()` to skip the rebuild/delete/copy.)

Might be able to solve this with a lateinit, but checking for null seems just as easy as checking for isInit.

Reply:
Resolved via conversation.